src/System.Management.Automation/FormatAndOutput/common/PSStyle.cs

```diff
@@ -259,6 +271,27 @@ public string FromRgb(int rgb)
             }
         }
 
+        /// <summary>
+        /// Contains configuration for the progress bar visualization.
+        /// </summary>
+        public class ProgressConfiguration
+        {
+            /// <summary>
+            /// Gets or sets the style for progress bar.
+            /// </summary>
+            public string Style { get; set; } = "\x1b[33;1m";
```

Comment:
Maybe add in the description that is the default.

Reply:
You mean describe in text what the ANSI escape sequence represents?